package.json

```diff
@@ -68,11 +69,17 @@
     "superagent": "^3.2.1",
     "through2": "^2.0.3"
   },
+  "optionalDependencies": {
+    "pelias-interpolation": "*"
```

Comment:
does it make sense to put a version number here? I'm guessing no, since I wonder if Greenkeeper could even handle keeping it up to date.